shared/src/price_estimation/gas.rs

```diff
@@ -0,0 +1,24 @@
+///! constants to estimate gas use in GPv2
+
+/// minimum gas every settlement takes
+pub const SETTLEMENT: u64 =
+    // initial tx gas
+    32_000 +
+    // isSolver
+    7365;
+
+/// gas per trade excluding c20 transfer
+pub const TRADE: u64 =
+    // computeTradeExecutions
+    35_000 +
+    // transferFromAccounts and transferToAccount overhead
+    2 * 3000 +
+    // overhead of one interaction
+    3000;
```

Comment:
This seems weird to use `2 * 3000` for `transferFromAccounts and transferToAccount overhead`. Don't we want to use `2 * ERC20_TRANSFER`?

Reply:
The reason I have done it this way is to split up the GP gas use from the ERC20 gas use. In the future we might be able to estimate erc20 gas use more accurately per token so it's nice to not have it part of this sum.